utils.groovy

```diff
@@ -221,17 +235,23 @@ def bump_builds_json(stream, buildid, arch, s3_stream_dir, acl) {
 // build on a different architecture.
 //
 // Available parameters:
+//    env: map of key/value pairs of environment variables to set
 //    expiration: string that represents a golang duration for how
 //                long the container should last (i.e. 4h, 30m)
 //    image: string that represents the container image to pull
 //    workdir: string that represents the in container working directory
 def createCosaRemoteSession(params = [:]) {
+    def env = params['env']
     def expiration = params['expiration']
     def image = params['image']
     def workdir = params['workdir']
+    def envArgs = ""
+    for (x in env) {
+        envArgs += "--env ${x.key}=${x.value} "
+    }
```

Comment:
What we do elsewhere is e.g.

```
envArgs = env.collect { k, v -> "--env ${k}=${v}" }
```

and then `${envArgs.join(" ")}` below.

Reply:
will do